automatic/nexus-repository/nexus-repository.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nexus-repository</id>
+    <title>Nexus Repository OSS</title>
+    <version>3.49.0.02</version>
+    <authors>Sonatype</authors>
+    <owners>chocolatey-community,DarwinJS</owners>
+    <summary>Free open source version of popular Nexus Repository for binary artifacts including first class Nuget support.</summary>
+    <description></description>
+    <projectUrl>http://www.sonatype.com/nexus-repository-oss</projectUrl>
+    <projectSourceUrl>https://github.com/sonatype/nexus-public</projectSourceUrl>
+    <docsUrl>http://books.sonatype.com/nexus-book/3.0/reference/index.html</docsUrl>
+    <tags>nexus-repository server freeware cross-platform maven nuget npm java rubygems docker eclipse p2 osgi obr apt yum rpm artifact bower ssl respository sonatype nexus admin</tags>
+    <copyright>Sonatype</copyright>
+    <licenseUrl>http://www.eclipse.org/legal/epl-v10.html</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>https://cdn.jsdelivr.net/gh/git@github.com:chocolatey-community/chocolatey-packages/icons/nexus-repository.png</iconUrl>
+    <releaseNotes>https://help.sonatype.com/repomanager3/release-notes</releaseNotes>
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="[1.1.0,2.0)" />
```

Comment:
It seemed sensible, if 2.0 introduced breaking changes we'd want to be conscious of upgrading past there. Not a problem for a while, but something to consider.

That said, I can remove it if that's preferred.

Reply:
If it can be removed, then that would be the best.
Unless we use any of the functions (I didn't notice any) provided by the extension, it shouldn't be needed.

If it is still needed, then yeah, remove the upper limit.

Reply:
I can't find any of the functions I see in the extension in the files, here. I'll remove the dependency!